application/frontend/controllers/MfaController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace frontend\controllers;
+
+use frontend\components\BaseRestController;
+use Sil\Idp\IdBroker\Client\IdBrokerClient;
+use yii\filters\AccessControl;
+use yii\helpers\ArrayHelper;
+use yii\web\BadRequestHttpException;
+
+class MfaController extends BaseRestController
+{
+
+    /**
+     * @var IdBrokerClient
+     */
+    public $idBrokerClient;
+
+    public function behaviors()
+    {
+        return ArrayHelper::merge(parent::behaviors(), [
+            'access' => [
+                'class' => AccessControl::className(),
+                'rules' => [
+                    [
+                        'allow' => true,
+                        'roles' => ['@'],
+                    ],
+                ]
+            ]
+        ]);
+    }
+
+    public function init()
+    {
+        parent::init(); // TODO: Change the autogenerated stub
```

Comment:
Does this `TODO` still need to be addressed?

Reply:
oh, no, that was an ide thing I didnt clean up